package.json

```diff
@@ -98,9 +101,12 @@
     "file-loader": "^3.0.1",
     "fork-ts-checker-webpack-plugin": "^1.3.4",
     "generate-changelog": "^1.7.1",
+    "glob": "^7.1.6",
     "jest": "^24.8.0",
     "jest-styled-components": "^6.3.1",
     "jsx-to-string": "^1.4.0",
+    "lodash": "^4.17.15",
+    "mocha": "^7.1.1",
```

Comment:
Can we do the test with just `jest`?

Reply:
Oh yes, can, this was not supposed to be there anymore ^^